common/environment/check/no_display.sh

```diff
@@ -0,0 +1 @@
+export QT_QPA_PLATFORM=offscreen
```

Comment:
I think it's better to `export QT_QPA_PLATFORM=${QT_QPA_PLATFORM:-offscreen}`, some template defined it to other value.